cmd/serviceusers/create.go

```diff
@@ -0,0 +1,171 @@
+package serviceusers
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/appgate/sdp-api-client-go/api/v17/openapi"
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	"github.com/appgate/sdpctl/pkg/filesystem"
+	"github.com/appgate/sdpctl/pkg/prompt"
+	"github.com/appgate/sdpctl/pkg/util"
+	"github.com/hashicorp/go-multierror"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func NewServiceUsersCreateCMD(f *factory.Factory) *cobra.Command {
+	opts := ServiceUsersOptions{
+		Config: f.Config,
+		API:    f.ServiceUsers,
+		In:     f.Stdin,
+		Out:    f.IOOutWriter,
+	}
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   docs.ServiceUsersCreate.Short,
+		Long:    docs.ServiceUsersCreate.Long,
+		Example: docs.ServiceUsersCreate.ExampleString(),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I think you can reduce indentation level in this function significant if you refactor and return earlier